Corona-Warn-App/src/main/res/layout/fragment_information.xml

```diff
@@ -9,26 +9,22 @@
         android:layout_height="match_parent"
         android:contentDescription="@string/information_title">
 
-        <include
+        <com.google.android.material.appbar.MaterialToolbar
             android:id="@+id/information_header"
-            layout="@layout/include_header"
-            android:layout_width="match_parent"
+            android:layout_width="0dp"
             android:layout_height="wrap_content"
-            app:icon="@{@drawable/ic_close}"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintBottom_toTopOf="@id/scrollview"
-            app:title="@{@string/information_title}" />
+            app:navigationIcon="@drawable/ic_close"
```

Comment:
can you use `style="@CWAToolbar.Close"`  here ?

Reply:
Sure!